etc/config-template.toml

```diff
@@ -209,7 +209,11 @@
 # bit smaller.
 # region-max-size = "144MB"
 # region-split-size = "96MB"
-
+# When the region's keys exceeds region-max-keys, we will split the region
+# into two which the left region's keys will be region-split-keys or a little
+# bit smaller.
+# region-max-keys = 1440000
+# region-split-keys = 960000
```

Comment:
Why it is 960000? Please give some calculations to explain it.

Reply:
According to the following factors:
1. The smallest length of the index is 40~50bytes while most of the index may bigger than this length. 
2. The length of record data is unforeseen(must be bigger than 50bytes).
3. We can scan 1'000'000 keys in one second.
4. The query costs more than 1 second is a slow query in `coprocessor`.
5. The number of keys in the region will be controlled by both `region-split-keys` and `region-split-size`.
5. The number of regions should be considered(we cannot tolerate too many regions)

If we consider 40~50 as the index's average length,  the region-split-keys should be 2'000'000 according to `region-split-size`, so we need 2 seconds to finish the scan(slow query).

And if we consider 960'000 as `region-split-keys`:
1. for the length of index smaller than 100 bytes, the most number of the keys should be 960'000, while for the length of data bigger than 100 bytes, it's also be controlled to less than 960'000. 
2. On the other side, the split size of the region would be controlled to bigger than 48MB(half of the default `region-split-size`), so the number of regions won't be too much.